lib/send_sms.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Send_SMS
+
+  attr_reader :credentials
+
+  def initialize
+    @credentials = Dotenv.load
+  end
+
+  def confirmation_text
+    sender
+  end
+
+
+  private
+
+  def sender
+    message1 = "Thank you!, your order was placed and "
+    message2 = "will be delivered before #{stamp_time}"
+
+    client = Twilio::REST::Client.new credentials["sid"], credentials["token"]
+
+    client.account.messages.create(from: credentials["TWILIO_NUMBER"], to: credentials["MY_NUMBER"],
```

Comment:
Metrics/LineLength: Line is too long. [100/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)